src/rules/declaration-block-trailing-semicolon/README.md

```diff
@@ -0,0 +1,55 @@
+# declaration-block-trailing-semicolon
+
+Require or disallow a trailing semicolon with a declaration block.
```

Comment:
I'd like a little explanation of what a "trailing semicolon" is, because I don't think most people know that they *could* leave off the last semicolon of a block by some standards.